slash-commands/utility/helpcenter.js

```diff
@@ -25,21 +25,25 @@ module.exports = {
       const HelpCenterMessage = new MessageEmbed()
         .setTitle('Codecademy Help Center Resources')
         .setColor('DARK_NAVY')
-        .addField(
-          'Codecademy Help Center',
-          'https://help.codecademy.com/hc/en-us'
+        .addFields(
+          {
+            name: 'Codecademy Help Center',
+            value: 'https://help.codecademy.com/hc/en-us',
+          },
+          {
+            name: 'Submit A Request',
+            value: 'https://help.codecademy.com/hc/en-us/requests/new',
+          },
```

Comment:
I believe this was removed for the non-slash version of the command by Fede in the past. It should also be removed for the slash command in both the plaintext and embed versions.